git_machete/client.py

```diff
@@ -459,7 +459,7 @@ def get_root_of(branch: LocalBranchShortName) -> LocalBranchShortName:
             # after removing the outer layer of childless merged branches.
             # This is rare enough, however, that we can pretty much ignore this corner case.
 
-        print(bold("Discovered tree of branch dependencies:\n"))
+        print(bold("Discovered tree of branch dependencies:"))
```

Comment:
Whoops this changes the output (an empty line will be missing in the output as an effect)